ci-operator/config/openshift/installer/openshift-installer-release-4.3.yaml

```diff
@@ -159,11 +159,10 @@ tests:
   commands: setup_ssh_bastion; TEST_SUITE=openshift/disruptive run-tests; TEST_SUITE=openshift/conformance/parallel run-tests
   openshift_installer:
     cluster_profile: aws
-- as: e2e-gcp-upgrade
-  commands: TEST_SUITE=all run-upgrade-tests
-  openshift_installer:
+- as: e2e-upgrade
```

Comment:
I don't really care about adjusting tests for 4.5 and earlier.  And upgrade jobs are useless for the installer anyway, because we're installing the target branch, and then updating to a release with the new installer code, so the code in flight with the PR is never exercised.  But if we do go back and touch these earlier releases without dropping the update jobs completely, I think we should consolidate around the single, multi-step, platform agnostic update job:

```console
$ git --no-pager grep 'context: .*upgrade' origin/pr/11662 ci-operator/jobs/openshift/installer
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:    context: ci/prow/e2e-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.1-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.2-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.3-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.3-presubmits.yaml:    context: ci/prow/e2e-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.4-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.4-presubmits.yaml:    context: ci/prow/e2e-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.5-presubmits.yaml:    context: ci/prow/e2e-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml:    context: ci/prow/e2e-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.7-presubmits.yaml:    context: ci/prow/e2e-aws-upgrade
origin/pr/11662:ci-operator/jobs/openshift/installer/openshift-installer-release-4.7-presubmits.yaml:    context: ci/prow/e2e-upgrade
```

But if we don't want to hold up this PR, I'm fine landing it as it stands and filing a follow up to drop all the 4.3+ e2e-aws-upgrade.  Thoughts?